src/helpers.ts

```diff
@@ -257,24 +257,24 @@ export function getDurationInUs(duration: number) {
   return Math.trunc(duration * 1e6);
 }
 
-export function megabytesToBytes(megabytes: number) {
-  return megabytes * 1024 * 1024;
+export function megabitsToBytes(megabytes: number) {
+  return megabytes * 125000;
 }
 
-export function bytesToMegabytes(bytes: number) {
-  return bytes / 1024 / 1024;
+export function bytesToMegabits(bytes: number) {
+  return bytes / 125000;
 }
 
 export const DEFAULT_NETWORK_CONDITIONS: NetworkConditions = {
-  downloadThroughput: megabytesToBytes(5), // megabytes/second
-  uploadThroughput: megabytesToBytes(3), // megabytes/second
+  downloadThroughput: megabitsToBytes(5), // megabytes/second
+  uploadThroughput: megabitsToBytes(3), // megabytes/second
   latency: 20, // milliseconds,
   offline: false,
 }
 
 export function formatNetworkConditionsArgs(networkConditions: NetworkConditions) {
```

Comment:
Please add comment saying that we need Devtools CDP expects format to be in bytes/second to make it clear.